lib/batch.js

```diff
@@ -35,6 +35,8 @@ internals.mergeResponses = (tableName, responses) => {
 internals.paginatedRequest = (request, table, callback) => {
   const responses = [];
 
+  const moreKeysToProcessFunc = () => request !== null && !_.isEmpty(request);
```

Comment:
Moved this up here so that i could re-use it to determine if I should add RequestItems